src/inputs/input-with-icon.scss

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import './input';
+
+// Mixin to have inputs combined with icon (eg. select)
+@mixin iui-input-with-icon {
+  @include iui-input;
+  display: grid;
+  align-items: center;
+  grid-template: 'input icon' / 1fr auto;
+  margin: 0;
+  padding: 0;
+  vertical-align: baseline;
+  position: relative;
+  width: 100%;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: border-color $iui-speed-fast ease-out;
```

Comment:
This doesn't need the motion media query because it's just transitioning a color in-place

Reply:
removed media query